test/test_images/performance/main.go

```diff
@@ -112,12 +115,17 @@ func main() {
 		exec = newSenderReceiverExecutor(pacerSpecs, aggCli)
 
 	case "aggregator":
+		var tags []string
+		if makoTags != "" {
+			tags = strings.Split(makoTags, ",")
+		}
```

Comment:
You could replace these 4 lines with just
```golang
tags := strings.Split(makoTags, ",")
```

Reply:
The problem is that if the `makoTags = ""`, this will result in an array like this `[""]`

Reply:
Right, my mistake.